web/src/features/prompts/server/actions/createPrompt.ts

```diff
@@ -84,5 +87,15 @@ export const createPrompt = async ({
 
   const [createdPrompt] = await prisma.$transaction(create);
 
+  try {
+    EventsBroker.dispatch(brokerEvents.BrokerEventPromptCreated.create({
+      payload: {
+        object: LangfusePrompt.ofStruct(createdPrompt)
+      }
+    }))
+  } catch (error) {
+    Sentry.captureException(error);
+  }
```

Comment:
This would become the caller DX. Note: type drifts here would become noticeable to the prompt maintainer. However, currently, due to the overall type drift, some properties are patched in the event object payload. 

Otherwise prisma would e.g. leave unknown, requiring the caller to cast. If the caller passed invalid type the toObject call in EventBroker will fail via zod anyways.